### PR TITLE
#6 Better Readme. updated version

### PR DESCRIPTION
**Pull request by andressbarajas**
Pull request number 6 from Sizious dcload-serial Repo. Contains commits from:

https://github.com/sizious/dcload-serial/commit/eed71056d8dfc04e6d9138ebbaf612b3de1ce778

to

https://github.com/sizious/dcload-serial/commit/a2b071e8f140ca7fe85cb8122392b209a0892d36

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,17 @@
+WHAT'S NEW IN 1.0.6
+
+* Complete redesign of the build system and compatibility improvements for
+  various OS, including: BSD (FreeBSD), GNU/Linux (Debian/Ubuntu), macOS, 
+  MinGW/MSYS (i.e. DreamSDK), MinGW-w64/MSYS2 and Cygwin. The build system is
+  now shared between dcload-ip and dcload-serial. [SiZiOUS]
```
Comment (ljsebald) [inline CHANGES]:
```suggestion
  MinGW/MSYS (i.e. DreamSDK), MinGW-w64/MSYS2 and Cygwin. [SiZiOUS]
```

File: Makefile.cfg [hunk 2]
```diff
@@ -12,7 +12,6 @@ HOSTLDFLAGS		= -L/usr/local/lib
 # For MinGW/MSYS, MinGW-w64/MSYS2 and Cygwin
 ifdef WINDOWS
   HOSTCFLAGS += -D_WIN32
-  HOSTLDFLAGS += -static
```
Comment (ljsebald) [inline Makefile.cfg]: This is undoing something in a previous PR. Remove.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
+# dcload-serial 1.0.6
+
+**dcload** is a **Sega Dreamcast** (DC) serial loader written originally by 
+[Andrew Kieschnick](http://napalm-x.thegypsy.com/andrewk/dc/), a.k.a.
+**ADK/Napalm**. This program is part of 
+[KallistiOS](http://gamedev.allusion.net/softprj/kos/) (KOS).
```
Comment (ljsebald) [inline README.md]:
```suggestion
**ADK/Napalm**. It is currently maintained as a part of 
[KallistiOS](http://gamedev.allusion.net/softprj/kos/) (KOS).
```

File: make-cd/Makefile [hunk 4]
```diff
@@ -18,7 +21,7 @@ burn-audio: audio.raw
 	touch burn-audio
 
 1st_read.bin: $(1ST_READ)
-	$(SCRAMBLE) $(1ST_READ) 1st_read.bin
+	cp $(1ST_READ) 1st_read.bin
```
Comment (ljsebald) [inline make-cd/Makefile:25]: Why did the call to scramble get taken out? This will produce a non-bootable disc as it is.

File: Makefile.cfg (hunk 2 above)
Comment (andressbarajas) [reply Makefile.cfg]: Removed

File: make-cd/Makefile (hunk 4 above)
Comment (andressbarajas) [reply make-cd/Makefile:25]: Brought back scramble

**Pull request merged**